src/components/DpAutocomplete/DpAutocomplete.vue

```diff
@@ -1,37 +1,67 @@
 <template>
   <div class="relative">
+    <!-- Label displayed on small screens if search button is used -->
+    <label
+      v-if="searchButton"
+      id="autocomplete-label"
+      for="input-box"
+      class="mb-1 md:hidden">
+      {{ placeholder }}
+    </label>
     <!-- Search container -->
     <div
       aria-haspopup="listbox"
       aria-controls="options-list"
       :aria-expanded="isOptionsListVisible"
       :class="[
         isInputFocused ? 'border-interactive' : 'border-gray-300',
+        searchButton && isPlaceholderVisible ? 'justify-between' : 'justify-end',
         { 'focus-within:ring-1 focus-within:ring-interactive': isInputFocused }
       ]"
-      class="border rounded-md flex items-center overflow-hidden relative transition-colors duration-200 ease-in-out px-1 py-[2px]"
+      class="border rounded-md flex items-center overflow-hidden relative transition-colors duration-200 ease-in-out"
       role="combobox"
       :style="boxHeightStyle"
       tabindex="0"
       @mousedown.stop.prevent="focusInput">
+      <!-- Placeholder, if search button is used only displayed on lap-up screens -->
       <label
         id="autocomplete-label"
         for="input-box"
         :class="{
-          'h-full w-full px-1 py-[2px] text-gray-400 flex items-center m-0': true,
-          'absolute pointer-events-none': !isPlaceholderVisible,
+          'h-full w-full px-1 py-[2px] text-gray-400 items-center m-0': true,
+          'flex': !searchButton,
+          'hidden md:flex': searchButton,
+          'md:absolute md:pointer-events-none': !isPlaceholderVisible,
           'sr-only': !isPlaceholderVisible
         }">
         {{ placeholder }}
       </label>
+      <!-- Placeholder for small screens if search button is used -->
+      <template v-if="searchButton">
+        <span
+          :class="{ 'hidden': !isPlaceholderVisible }"
+          class="md:hidden h-full w-[80%] px-1 py-[2px] text-gray-400 flex items-center m-0">
+        {{ de.search.searching }}
+      </span>
```

Comment:
```suggestion
        </span>
```

Reply:
Thanks, done in 1c87298fe3fa2eb2e9e9325eda7b4188ff8a7743